### DISCUSSION TITLE
Revert revision per site

### DISCUSSION THREAD

**Discussion by carolineboeykens** [craftcms/cms#8888] [Ideas] [answered]
### Description
We would like to revert a revision for an entry in a specific site and that the entry in the other sites stay unchanged.

### Steps to reproduce
1. Create channel with propagation method: "Save entries to all sites enabled for this section" and set the fields on this section translatable per site.
2. Create a new entry and fill the fields with content.
3. Change the content in site A
4. Change the content in site B
5. Revert the last revision in site A
6. The entry in site A and site B is reverted

**Comment by sidedwards** [comment]
@brandonkelly it seems like this is still an issue in localization workflows. Is there a recommended workaround for this scenario?

**Comment by brandonkelly** [comment, marked as answer]
It’s not really possible to just revert one site’s content, since all sites’ content is interconnected, per the fields’ Propagation Method settings.

More likely, we will implement some sort of content comparison view (#8135), where you could compare the Current revision to a prior one, and pull in specific changes from it. So not really a full _revert_; just an automated content copy+paste.

**Reply by sidedwards** [reply]
Thanks for the info; that makes sense.

A comparison tool to pull site-specific content from a revision into a draft would be great for multi site editing workflows where  revisions can't be used without also reverting all site content.

We recently included a feature in the Translations plugin to [select an Entry draft as a source](https://github.com/AcclaroInc/craft-translations/discussions/224). We could also include support for revisions, but it would really only be useful for translations workflows.
